### ISSUE TITLE
Error : InvalidKeyOrUnauthorizedURLMapError

### ISSUE THREAD

**Issue by Basems** [dburles/meteor-google-maps#104]
We use dburles:google maps for allowing Google auto-complete search (for addresses), but the following error message always prompts

"This page was unable to display a Google Maps element. The provided Google API key is invalid or this site is not authorized to use it. Error Code: InvalidKeyOrUnauthorizedURLMapError"

We ensured that our browser key has been generated correctly as recommended by Google doc. and we added all possible referrers. However, the same error message still appears. 

Any pointers what's wrong?

**Comment by dburles** [comment]
Hey @Basems did you end up resolving this one? It's not something I've come across before, are you able to make use of the key by manually loading the maps library (paste the script tag in your application `<head>` tags)

**Comment by dburles** [comment]
That should help determine whether it's an issue with this package or something amiss with the configuration

**Comment by Basems** [comment]
Hi David,

Thanks for your follow up and suggestions.

We did fix the issue. It was the way we specify the key in meteor start up.
We specified it in the settings.js file

We used to specify it this way: GoogleMaps.load({v: "3", key:
Meteor.settings.googleMap_key, libraries: 'places'});

Instead we directly specified the key in the function call
Meteor.startup(function() {
  GoogleMaps.load({
    key: "API KEY",
   ...
